format/spec.md

```diff
@@ -513,6 +514,17 @@ Manifests for a snapshot are tracked by a manifest list.
 
 Valid snapshots are stored as a list in table metadata. For serialization, see Appendix C.
 
+Statistics files' metadata within `statistics` field is a struct with the following fields:
+
+| Field name                      | Type                               | Description                                                                                                                                                                                                              |
```

Comment:
Can you add the version columns to this table so it is clear that this is optional?

Reply:
It's versioned and marked as optional the "A snapshot consists of the following fields" table.

Within "Statistics files' metadata" all fields are required. How should i indicate that?

Reply:
Are these field required or optional?

Reply:
Here, all fields are required.

Reply:
actually no, the newly added properties field is optional. i added a column in the table to indicate that